src/chat/chatter.py

```diff
@@ -183,12 +183,27 @@ def update(self):
         # Status icon handling
         url = client.instance.urls.get(player.login)
         if url:
-            if url.scheme() == "fafgame":
-                self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
-            elif url.scheme() == "faflive":
-                self.statusItem.setIcon(util.icon("chat/status/playing.png"))
-                self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
+            if int(url.queryItemValue("uid")) in client.instance.games.games:
+                game = client.instance.games.games[int(url.queryItemValue("uid"))]
```

Comment:
I still don't like the way we get the uid here, but from what I've seen browsing the code lately, the very way urls find their way into here is quite a mess in itself. It can probably work for now, especially given that we pull things from the url right above it anyway. One day we'll probably refactor the url stuff.